app/src/main/java/org/intelehealth/app/activities/notification/view/NotificationAdapter.kt

```diff
@@ -79,13 +80,15 @@ class NotificationAdapter(
     inner class MyHolderView(itemView: View) : RecyclerView.ViewHolder(itemView) {
```

Comment:
I really appreciate that you have designed class in Kotlin, you can also use the Data Binding to reducing the initialization of view component